pkg/cassandra/datacenter.go

```diff
@@ -124,6 +125,20 @@ type DatacenterConfig struct {
 	InitialTokensByPodName map[string][]string
 }
 
+// SanitizedName returns a sanitized version of the name returned by CassClusterName()
+func (in *DatacenterConfig) SanitizedName() string {
```

Comment:
I don't see this getting called anywhere, yet the CRD description says it's used to name Kubernetes objects.

Reply:
Indeed, I think I initially imagined I would need the sanitized name, but in the end I only need the non sanitized version of the datacenter name to populate the env variable.
I'll remove the function.